barcode/codex.py

```diff
@@ -44,11 +44,11 @@ class Code39(Barcode):
     """
 
     name = 'Code 39'
+    add_checksum_default = True
```

Comment:
Moving this to the constructor seems more intuitive (since it's an instance attribute, and not a class attribute).
Also, should probably be named `add_checksum`, since this isn't the default, but the current value.